lib/streamlit/hello/util.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018-2019 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAX_DOCSTRING = 1000
+
+
+def remove_declaration_and_docstring(lines):
+    """
+    This function parses the source code of a function and removes the function
```

Comment:
Use Numpy doc style. The first sentence must be a summary that fits in one line. And shouldn't say "This function" or use present tense "parses". Instead just use the imperative "parse".

Example:
> """Return a function's source code with the docstring removed.
> 
> This function parses the source code (...)
> """